core/assets/bundles/bundle.properties

```diff
@@ -48,6 +48,7 @@ customgame = Custom Game
 newgame = New Game
 none = <none>
 minimap = Minimap
+position = Position
```

Comment:
Unless I'm missing something, this is unnecessary.

Reply:
variable name consistency

Reply:
What? I'm saying this string isn't used anywhere in the PR, or in the base game, as far as I'm aware.